site/src/pages/_app.tsx

```diff
@@ -18,6 +18,12 @@ import { AppProps } from 'next/app'
 type ComponentWithLayout = AppProps['Component'] & {
   getLayout?: (page: JSX.Element) => JSX.Element
 }
+;(async () => {
```

Comment:
What if we just import this styles as usual, but on a particular `careers` page?

Reply:
https://nextjs.org/docs/messages/css-global

Reply:
I am pretty sure if i will visit a page from main page i've got a not a style. You should check url in another way.

Reply:
It will work, because page was reloaded on careers page, but I did refactoring to solve it another way